README.md

```diff
@@ -11,6 +11,17 @@
   * Utilities to interact with EC2 instances
   * s3 up/download
 
+* Dropbox
+ * API v1 wrappers
+
+## Config setup
+
+Export environment variable(s) (or set on command line) for EDN format configuration file location.
+
+Set environment variable to override default configuration file location:
+
+    $ export LIB_CLJ_INFRA_CONFIG_FILE_PATH         "$HOME/conf/bd/common/clj/secret/clj-infrastructure.config.secret.edn"
```

Comment:
Well, I picked the name based on presence of clj-foundation which will likely want to use an env var, so having them both use LIB_CLJ prefix would make sense for making those two vars more obviously related.  Then (with code changes) I want LIB_CLJ_COMMON_CONFIG_[FILE_]_PATH that will work with both of them.  I will reconsider name...